.nuget/NuGet.targets

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is not needed - we have switched to using the DotNet CLI

Reply:
looks like a merge issue it shouldn't be there will fix it